swiftly/client/standardclient.py

```diff
@@ -558,10 +560,7 @@ def request(self, method, path, contents, headers, decode_json=False,
                     value = resp.read()
                     resp.close()
             except Exception as err:
-                status = 0
-                reason = '%s %s' % (type(err), str(err))
-                hdrs = {}
-                value = None
```

Comment:
> It'd mean the verbose call, conn.close(), and reset_func would no longer be called

Ideally, more specific exceptions should be captured in the places where doing those things is appropriate.

Catch-all-and-effectively-ignore makes debugging trickier. :S

> This would change behavior

Indeed, and not an appropriate thing to do when just attempting to fix python3 compatibility. I'll remove this change.